mmv1/third_party/terraform/tpgresource/utils.go

```diff
@@ -666,6 +673,13 @@ func BuildReplacementFunc(re *regexp.Regexp, d TerraformResourceData, config *tr
 		}
 	}
 
+	if strings.Contains(linkTmpl, "{{universe_domain}}") {
```

Comment:
Under what conditions will this variable appear in a resource URL? Depending on the answer we may need to document those conditions, or add a test to exercise them.

Reply:
I remember this being implemented when I tried to change the basePath for the product. The goal was to replace the universe field by it's appropriate value in the long URL. (As some IAM tests were having long URL for their IDs). I'm not sure if these lines are still being used after replacing the base paths in the current implementation.

Reply:
Let's remove it in that case! As-is it's dead code that might negatively surprise us in the future.

Reply:
Sure, I'll test if it's safe to do so, if yes I'll take this part out.